syntax = "proto3";

// The greeting service definition.
service API {
    // Send a query response
    rpc processRequest (apiRequest) returns (apiResponse) {}
  }
  
  // The request message containing the symbol and investment amount
  message apiRequest {
    string symbol = 1;
    int32 investment = 2;
  }
  
  // The response message containing the result message and graph_data
  message apiResponse {
    string message = 1;
    string graph_data = 2;
  }